src/components/Builds.vue

```diff
@@ -30,7 +30,7 @@
           </div>
           <div class="extra content">
             <span class="left floated hashtag build-id">
-              <a target="_blank" v-bind:href="build.link_to_branch">
+              <a target="_blank" v-bind:href="build.link_to_build">
```

Comment:
the template is calling "link_to_build". Not sure if "link_to_pipeline" will be called or not.
if not, could you please atleast make it configurable - IMO many people use such dashboards to verify the last successful pipeline numbers and want to keep them handy.

Thanks for the help in advance.
-Vijay

Reply:
link_to_build is set in the src/main.js file. It is either the link to the pipeline or the job, it depends on the apiVersion. I use this modified version myself, it gives me direct access to the last pipeline/job.